src/js/weather.js

```diff
@@ -1,74 +1,67 @@
+                    lng: `${lng}`
+                }
+            },
+            options: {
+                thumbMaps: false
             }
-        },
-        options: {
-            thumbMaps: false
         }
-    });
-    var xhr = new XMLHttpRequest();
-    xhr.open("POST", url);
-    xhr.responseType = 'json';
-    xhr.setRequestHeader("Content-Type", "application/json");
-    xhr.onreadystatechange = function () {
-        if (xhr.readyState === 4) {
-            console.log(xhr.status);
-            console.log(xhr.response);
-        }
-    };
-    xhr.send(data);
-    xhr.onload = function () {
-        z = xhr.response.results[0].locations[0].adminArea5 
-        str = JSON.stringify(z)
-        deleteSymbols = str.split('"').join('')
-        console.log(data);
-    }
+      })
+      
+    .then(function (response) {
+        city = response.data.results[0].locations[0].adminArea5
+        cityStr = JSON.stringify(city)
+        deleteSymbols = cityStr.split('"').join('')
+        
+        
+    })
+    .catch((error) =>{
+        console.error(error);
+        
+    })
+    
 }
 function ShowLocalWeather() {
-    fetch(`http://api.openweathermap.org/data/2.5/weather?q=${deleteSymbols}&appid=add83517209f776fcced4e6690e72a82`)
-        .then(function (resp) { return resp.json() })
-        .then(function (data) {
-            console.log(data);
-            document.querySelector('.city').innerHTML = data.name;
-            document.querySelector('.temp').innerHTML = Math.round(data.main.temp - 273) + " ℃";
-            document.querySelector('.status').innerHTML = data.weather[0].description;
-            document.querySelector('.wind').innerHTML = "скорость ветра " + data.wind.speed;
-            document.querySelector('.icon-weather li').innerHTML = `<img scr = "http://openweathermap.org/img/wn/${data.weather[0].icon}@2x.png">`;
-        })
-        .catch(function () { })
+    showKiev(deleteSymbols)
 }
 function ShowKievWeather() {
-    fetch(`http://api.openweathermap.org/data/2.5/weather?q=Kiev&appid=add83517209f776fcced4e6690e72a82`)
-        .then(function (resp) { return resp.json() })
-        .then(function (data) {
-            console.log(data);
-            document.querySelector('.city').innerHTML = data.name;
-            document.querySelector('.temp').innerHTML = Math.round(data.main.temp - 273) + " ℃";
-            document.querySelector('.status').innerHTML = data.weather[0].description;
-            document.querySelector('.wind').innerHTML = "скорость ветра " + data.wind.speed;
-            document.querySelector('.icon-weather li').innerHTML = `<img scr = "http://openweathermap.org/img/wn/${data.weather[0].icon}@2x.png">`;
-        })
-        .catch(function () { })
+    showKiev('Kiev')
+}
+export function showKiev(urlparamter) {
+    fetch(`http://api.openweathermap.org/data/2.5/weather?q=${urlparamter}&appid=add83517209f776fcced4e6690e72a82`)
+    .then(function (resp) { return resp.json() })
```

Comment:
Use arrow function `(response) => {...}` here and everywhere.
Don't cut words (variable names, parameters etc.) without reason.